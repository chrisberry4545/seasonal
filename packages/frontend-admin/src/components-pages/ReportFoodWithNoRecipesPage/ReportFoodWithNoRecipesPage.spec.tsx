import React from 'react';
import { ReportFoodWithNoRecipesPage } from './ReportFoodWithNoRecipesPage';
import { shallow, ShallowWrapper } from 'enzyme';

jest.mock('../../components-main', () => ({
  ReportFoodWithNoRecipes: () => 'ReportFoodWithNoRecipes'
}));

describe('<ReportFoodWithNoRecipesPage />', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() =>
    wrapper = shallow(
      <ReportFoodWithNoRecipesPage />
    )
  );

  test('renders correctly', () => expect(wrapper).toMatchSnapshot());

});
