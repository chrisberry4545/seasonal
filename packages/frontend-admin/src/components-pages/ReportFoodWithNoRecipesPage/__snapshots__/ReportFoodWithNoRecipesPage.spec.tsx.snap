// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReportFoodWithNoRecipesPage /> renders correctly 1`] = `<ReportFoodWithNoRecipes />`;
